projects/player/modules/verona/services/verona-subscription.service.ts

```diff
@@ -34,7 +34,8 @@ export class VeronaSubscriptionService {
   private handleMessage(messageData: VopMessage): void {
     switch (messageData.type) {
       case 'vopStartCommand':
-        LogService.info('player: _vopStartCommand ', messageData);
+        LogService.debug('player: _vopStartCommand ', messageData.type, messageData.sessionId);
+        LogService.debug('player: _vopStartCommand unit definition ', messageData.unitDefinition);
```

Comment:
Hier habe ich aufgeteilt, weil man die unit def als JSON-String eh nicht sinnvoll lesen konnte.

Reply:
Wichtig ist hier eigentlich, ob die Message vopStartCommand kommt und welche Parameter sie hat. Die unitDefinition wird an anderer Stelle in der UnitComponent ausgegeben  - ist also hier überflüssig. Wenn die Ausgabe der gesamten message unter FF störend ist, kann meinetwegen auch nur "vopStartCommand" ausgegeben werden.